srm.js

```diff
@@ -25,7 +25,36 @@ function rethrowError(err) {
     throw parseError(err)
 }
 
-class CodeDxApiClient {
+function checkIfBranchPresent(branches, branchName) {
+    let result = false
+    branches.forEach(branch => {
+        if (branch.name.includes(branchName)) {
+            result = true
+        }
+    })
+    return result
```

Comment:
Can use `for (x of y)` to simplify this without needing to explicitly index into each element:

```js
for (branch of branches) {
    if (branch.name.includes(branchName)) return true
}
return false
```

Reply:
Though the `.includes` check should just be `==`